DEPLOYMENT.md

```diff
@@ -0,0 +1,176 @@
+## ODM2 RESTful Web Services -- Deployment Guide
+
+A Python RESTful web service inteface for accessing data in an ODM2 database via Django rest swagger APIs.
+
+NOTE: Currently this repository is under heavy development, a working version is not guaranteed at any point. We will start creating tagged releases soon; once available, please use the latest tagged release for stability.
+
+## Production Installation:
```

Comment:
Top title should be `# ODM2 Restful Web Services -- Production Deployment Guide`

Reply:
Got it.